### DISCUSSION TITLE
Prioritizing topics for TRAPI 1.2 by vote

### DISCUSSION THREAD

**Discussion by edeutsch** [NCATSTranslator/ReasonerAPI#251] [Ideas]
This thread will serve as the voting platform for prioritizing the topics to tackle for TRAPI 1.2. After everyone votes, we will address them roughly in order of acquired votes.

You are NOT voting for the PRs as written.
You ARE voting to prioritize the topics to be resolved and included for August 1 TRAPI 1.2 release!

Vlado's Voting Scheme:
+2 Highest (heart) - Must get this done for TRAPI 1.2 August 1st
+1 High (thumb up) - Should try to get this done for TRAPI 1.2 August 1st
0 Medium (none) - Meh, if there's extra time by August 1st, sure
-1 Low Priority (thumb down) - This should not be prioritized. TRAPI 1.3 is better.

Vote this comment if you like the system!

**Comment by edeutsch** [comment]
Batch query support
( this refers to extending the already rudimentary "ids" "categories" batch support )

- https://github.com/NCATSTranslator/ReasonerAPI/issues/145
- https://github.com/NCATSTranslator/ReasonerAPI/issues/124
- Vote with emoji: heart, thumbs up, nothing, thumbs down

**Reply by edeutsch** [reply]
DONE. Decided we didn't need it.

**Comment by edeutsch** [comment]
Asynchronous query mechanisms (including callback)

- https://github.com/NCATSTranslator/ReasonerAPI/pull/147
- https://github.com/NCATSTranslator/ReasonerAPI/pull/150
- https://github.com/NCATSTranslator/ReasonerAPI/issues/96
- Vote with emoji: heart, thumbs up, nothing, thumbs down

**Reply by edeutsch** [reply]
DONE.

**Comment by edeutsch** [comment]
Support for workflow[ ]

- https://github.com/NCATSTranslator/ReasonerAPI/pull/200
- https://github.com/NCATSTranslator/ReasonerAPI/issues/152
- this is still a draft
- Vote with emoji: heart, thumbs up, nothing, thumbs down

**Reply by edeutsch** [reply]
DONE.

**Comment by edeutsch** [comment]
Add 'score' property to EdgeBinding

- https://github.com/NCATSTranslator/ReasonerAPI/pull/181
- Vote with emoji: heart, thumbs up, nothing, thumbs down

**Comment by edeutsch** [comment]
Enable specification of QEdge "length" (N-M intermediate hops)

- https://github.com/NCATSTranslator/ReasonerAPI/issues/154
- Vote with emoji: heart, thumbs up, nothing, thumbs down

**Comment by edeutsch** [comment]
Multiple Queries (AND/OR them together)

- https://github.com/NCATSTranslator/ReasonerAPI/issues/105
- Vote with emoji: heart, thumbs up, nothing, thumbs down

**Comment by edeutsch** [comment]
Align Attribute and QueryContraint
- https://github.com/NCATSTranslator/ReasonerAPI/issues/229

**Comment by vdancik** [comment]
Support for nested attributes

Vote with emoji: heart, thumbs up, nothing, thumbs down

**Comment by edeutsch** [comment]
Develop a mechanism to advertise which Attribute types and Constraint types are supported by a resource

* #257 

Vote with emoji: heart, thumbs up, nothing, thumbs down
